app-config.yaml

```
@@ -99,6 +99,10 @@ catalog:
     - type: url
       target: https://github.com/backstage/backstage/blob/master/catalog-info.yaml
 
+    # Kubernetes demo component
+    - type: url
+      target: https://github.com/mclarke47/dice-roller/blob/master/catalog-info.yaml
```

Is there a good reason not to use the example in the main repo for this?
```suggestion
      target: https://github.com/backstage/backstage/blob/master/plugins/kubernetes-backend/examples/dice-roller/catalog-info.yaml
```

---

I added the `.../kubernetes-namespace` annotation so I could limit the RBAC necessary